packaging/linux/openra-join-servers.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=OpenRA
+GenericName=Real Time Strategy Game
+Comment=Reimagining of early Westwood Games
+Icon=openra
+Exec=openra Launch.Connect=%U
+Terminal=false
+StartupNotify=false
```

Comment:
Have you tried logging to see what arguments you are actually passing to the game?

If Launch.Connect is given a too-bogus argument then the game will silently ignore it and go straight to the shellmap.